src/utils/logging_loss.jl

```diff
@@ -32,11 +32,11 @@ Arguments:
 """
 function lossfn(HM::LuxCore.AbstractLuxContainerLayer, x, (y_t, y_nan), ps, st, logging::LoggingLoss)
     targets = HM.targets
-    ŷ, y, y_nan = get_predictions_targets(HM, x, (y_t, y_nan), ps, st, targets)
+    ŷ, y, y_nan, st = get_predictions_targets(HM, x, (y_t, y_nan), ps, st, targets)
     if logging.train_mode
-        return compute_loss(ŷ, y, y_nan, targets, logging.training_loss, logging.agg)
+        return compute_loss(ŷ, y, y_nan, targets, logging.training_loss, logging.agg), st
```

Comment:
`st` ? here, could we keep only a single value as the output?

Reply:
EDIT: Oh! it looks like this is the recommended way to do our custom loss functions :/ https://lux.csail.mit.edu/stable/api/Lux/utilities#Loss-Functions 

some other models will need updates.